base/statefulset.yaml

```diff
@@ -123,14 +123,14 @@ spec:
               path: "/health"
               port: http
               scheme: HTTPS
-            initialDelaySeconds: 30
+            initialDelaySeconds: 300
```

Comment:
Can you explain what this is fixing/why this value is better than the old one? So that in the future we know how to avoid regressions here

Reply:
https://cbea.ms/git-commit/#why-not-how :pray: